index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item)
+    this.items.sort((a, b) => a - b) 
+    this.length++
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.items.length) {
+      throw new Error('OutOfBounds')
+    } else {
+      return this.items[pos]
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length === 0) {
+      throw new Error('EmptySortedList');
+    } else {
+      return this.items.sort((a, b) => b - a)[0]
+    }
+  }
 
-  min() {}
+  min() {
+    if (this.items.length === 0) {
+      throw new Error('EmptySortedList');
+    } else {
+      return this.items.sort((a, b) => a - b)[0]
+    }
+  }
 
-  sum() {}
+  sum() {
+    if(this.items.length === 0) {
+      return 0
+    } else {
+      return this.items.reduce((a, b) => a + b)
```

Comment:
```suggestion
      return this.items.reduce((a, b) => a + b, 0)
```

El metodo reduce necesita que se le especifique un valor por defecto, en esta caso al tratarse de un numero el valor por defecto que tu quieres es 0, entiendo que te pasen los test, pero si se empieza a complicar el asunto con el reduce si no lo especificas se nos hunde el barco :D